### PR TITLE
Revert broken changes, do not join.

### PR DESCRIPTION
**Pull request by timkelty**
@eliperelman changes introduced here broke things a bit: https://github.com/mozilla-neutrino/neutrino-dev/commit/9380f02e983e05e2780bb0a5a14e490d4a45891a#diff-3813a5588840cf846f7dfcf2cc0bc30b

This MW basically assumes you're using `image-loader`, or something else that would define image rules. Those pre-existing rule ids are what is configurable via `options.rules`.

For that reason, I was intentionally omitting the `.test`, as we can assume it already exists in that rule.

I'm then mapping the value of the actual test, so I can pass it all to the plugin's `test` option.

Your changes were using `options.rules` as extensions, and calling `.test`, which ended up passing something like `/(\.svg|\.img)$/`

Note:
I _was_ `join`ing those rules on a pipe for a regex string, which I didn't need to because `ImageminWebpackPlugin`'s test option can be an array.

**TL;DR**

This PR reverts those changes with the exception of the unnecessary `join('|')`. If you have other ideas or preferences how to do this, I'm happy to make changes.

### CODE REVIEW

File: packages/imagemin/index.js [hunk 1]
```diff
@@ -27,22 +27,20 @@ module.exports = (neutrino, opts = {}) => {
...
-      .test(test)
       .use(options.useId)
         .loader(imageminLoader)
         .options(options.imagemin);
 
-    return test;
-  });
+    return neutrino.config.module.rule(ruleId).get('test');
+  })
+  .filter(test => !!test);
```
Comment (eliperelman) [inline packages/imagemin/index.js]: This can be `.filter(Boolean)`.

Comment (timkelty) [reply packages/imagemin/index.js]: Of course. Old habit 🙈

**Comment by timkelty** [comment]
> I changed this during one of my refactors, I think for create-project tests, because I couldn't get some tests to pass. Maybe this is gone now though and we can move forward. Have you tested that this functions properly?

Yeah - actually I was testing other stuff w/ create-project and thats how I discovered it _wasn't_ working. I will do some more testing to be certain.

**Comment by timkelty** [comment]
Working in my tests
